src/markdown/utils.js

```diff
@@ -72,6 +73,12 @@ function escapeURL(str) {
  * @return {String}
  */
 function unescapeURL(str) {
+    // If URL is absolute, we shouldn't try to decode it
+    // since it doesn't represent a file path but rather a static resource
+    if (isAbsoluteURL(str)) {
+        return str;
```

Comment:
I'm wondering if we should still do `unescapeWith(URL_REPLACEMENTS_UNESCAPE, decoded)`

otherwise an url with `]` may return false positive when serializing.

Reply:
The escape of the url should also be updated to match this unescape.